packages/web/src/scss/components/Card/README.md

```diff
@@ -0,0 +1,398 @@
+content. This option works with both boxed and non-boxed Card, but is only available in the horizontal Card layout.
+
+```html
+<article class="Card Card--horizontal">
+  <div class="CardMedia CardMedia--auto CardMedia--filledHeight">
+    <!-- … -->
+  </div>
+  <div class="CardBody">
+    <!-- … -->
+  </div>
+</article>
+```
+
+ℹ️ Both options work with all media sizes.
+
+🎉 Fun fact: The `.CardMedia--expanded` and `.CardMedia--filledHeight` modifiers produce the same result for non-boxed
+horizontal (and reversed horizontal) Cards. But in all other contexts, the two modifiers behave differently.
+
+## CardLogo
+
+CardLogo is an optional subcomponent that displays a logo. To achieve the best visual result, use the PartnerLogo
+subcomponent.
+
+```html
+<div class="CardLogo">
+  <div class="PartnerLogo PartnerLogo--medium PartnerLogo--safeArea">
+    <img src="…" alt="Product Name" />
+  </div>
+</div>
+```
+
+## CardBody
+
+CardBody is the main content area of the Card.
+
+```html
+<div class="CardBody">
+  <!-- … -->
+</div>
+```
+
+### CardTitle
+
+CardTitle displays the main title of the Card. It uses the `<h4>` heading element by default, but you can use any other
+heading level that fits your document outline.
+
+```html
+<h4 class="CardTitle">
+  <a href="#">Card Title</a>
+</h4>
+```
+
+To emphasize the CardTitle, you can use the `CardTitle--heading` modifier:
+
+```html
+<h4 class="CardTitle CardTitle--heading">
+  <a href="#">Card Title</a>
+</h4>
+```
+
+👉 See below how to extend the link in CardTitle to [make the whole card clickable](#making-the-whole-card-clickable).
+
+### CardEyebrow
+
+CardEyebrow is an optional subcomponent that accompanies the CardTitle.
+
+```html
+<div class="CardEyebrow">Content options</div>
+<h4 class="CardTitle">Card Title</h4>
+```
+
+## CardFooter
+
+Use CardFooter for actions or any other content at the bottom of the Card. When using Cards with CardFooter in a Grid,
+the CardFooters will automatically line up.
+
+```html
+<footer class="CardFooter">
+  <!-- … -->
+</footer>
+```
+
+### Footer Alignment
+
+The footer can be horizontally aligned to the start, center, or end of the Card. To align the footer, use one of the
+following CSS modifiers:
+
+- `CardFooter--alignmentXLeft`
+- `CardFooter--alignmentXCenter`
+- `CardFooter--alignmentXRight`
+
+## Full Example
+
+When you put it all together:
+
+```html
+<article class="Card Card--vertical Card--boxed">
+  <div class="CardMedia CardMedia--auto">
+    <div class="CardMedia__canvas">
+      <img src="…" alt="" />
+    </div>
+  </div>
+  <div class="CardLogo">
+    <div class="PartnerLogo PartnerLogo--medium PartnerLogo--safeArea">
+      <img src="…" alt="Logo" />
+    </div>
+  </div>
+  <div class="CardBody">
+    <div class="CardEyebrow">Content options</div>
+    <h4 class="CardTitle CardTitle--heading">
+      <a href="#" class="CardLink">Card Title</a>
```

Comment:
it is weird that we use an example before we explain it. Maybe the example should be the last section?